int32 car_connected
int32 steer_degree
int32 steer_raw
int32 steer_limit_left
int32 steer_limit_center
int32 steer_limit_right
int32 rotor_hall
int32 rotor_hall_l
int32 rotor_hall_r
int32 voltage
int32 current_charging
int32 current_discharging
int32 drv_temperature
int32 car_vector
int32 car_lamp
int32 car_signal
int32 car_solidbrk
int32 embed_custom_a
int32 embed_custom_b